.eslintrc.json

```diff
@@ -3,9 +3,18 @@
     "node": true
   },
   "plugins": ["jsdoc"],
-  "extends": ["eslint:recommended", "plugin:jsdoc/recommended", "prettier"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:jsdoc/recommended",
+    "plugin:jest/recommended",
+    "plugin:node/recommended",
+    "prettier"
+  ],
   "globals": { "Set": true, "Symbol": true },
   "rules": {
+    // TODO
```

Comment:
Might be worth renaming .eslintrc.json to .eslintrc.js later so that the file is valid, but that's nitpicking since ESLint works with comments in JSON too.

BTW you could set it to 4.5.0 so that it works, but the discrepancy between the engines version and the current code will still exist.

Reply:
Yeah — we really need to find a solution for this. I've disabled the rule for now and will revisit it soon.